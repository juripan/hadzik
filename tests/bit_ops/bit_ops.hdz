naj x = 100 & 12 // 4
naj y = 30 | 17 // 31

vychod(x + y)