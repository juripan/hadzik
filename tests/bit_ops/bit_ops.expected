stdout: | stderr: | returncode: 35